webapp/src/app.tsx

```diff
@@ -3090,6 +3036,10 @@ export class ProjectView
         }
     }
 
+    hasCloudSync() {
+        return this.isLoggedIn();
```

Comment:
Note: maybe once we go to AAD we might not be able to store data ourselves and we might need OneDrive? Probably worth adding a TODO comment